include/libswiftnav/constants.h

```diff
@@ -34,7 +34,7 @@
 /** The official GPS value of Pi.
  * This is the value used by the CS to curve fit ephemeris parameters and
  * should be used in all ephemeris calculations. */
-#define GPS_PI 3.14159265358979323846
+#define GPS_PI 3.1415926535898
```

Comment:
I believe the GPS ICD document specifies a lower precision version. See: www.gps.gov/technical/icwg/.